test/unit/authorizer_test.rb

```diff
@@ -112,7 +112,7 @@ def setup
     domain1     = FactoryGirl.create(:domain)
     domain2     = FactoryGirl.create(:domain, :name => 'a-domain.to-be-found.com')
     domain3     = FactoryGirl.create(:domain, :name => 'another-domain.to-be-found.com')
-    domain4     = FactoryGirl.create(:domain, :name => 'be_editable.to-be-found.com')
+    domain4     = FactoryGirl.create(:domain, :name => 'be-editable.to-be-found.com')
```

Comment:
["Can subdomains have an underscore in it?"](http://stackoverflow.com/questions/2180465/can-hostname-subdomains-have-an-underscore-in-it)